pkg/yamlutil/parser.go

```diff
@@ -174,3 +176,28 @@ func (p *Parser) unmarshal(reader io.Reader) (*parsed, error) {
 func (p *Parser) buildConfigFromParsed(parsed *parsed, b Builder) error {
 	return b.BuildFromParsed(parsed.objects)
 }
+
+func ParseMultiYamlFile(fileName string) ([][]byte, error) {
```

Comment:
Given this is a generic YAML parsing function it ought to live in `/pkg/utils/yaml`. This package, contrary to its name, isn't a set of generic YAML utils - apologies for the confusion.